### ISSUE TITLE
[Feature Request] Plugin to Downmix from > 2 Channels to 2 Channels (Stereo)

### ISSUE THREAD

**Issue by andreas-glaser** [Unmanic/unmanic#376]
Any chance that someone clever could create a plugin that can downmix from 5.1/7.1 to Stereo?

Get Channel Count:
```shell
ffprobe -show_entries stream=channels -select_streams a:0 -of compact=p=0:nk=1 -v 0 "my-file.mp4"
```

Downmix
```shell
ffmpeg -i "my-file.mp4" -map 0:0 -map 0:1 -c:v:0 copy -c:a:0 aac -ac 2 "my-file-stereo.mp4"
```

**Comment by cattface** [comment]
I'd like to second this feature request - perhaps there is already a way to do it with custom commands but I've got no idea where to start.

 It would be really useful for me because I only have a stereo setup along with the other users of my Plex server. Re-encoding would save loads of server resources having to down mix on the fly, especially as Plex doesn't do a great job most of the time.

**Comment by DualScorch** [comment]
This can already be accomplished by using "Audio Encoder AAC", just enable custom ffmpeg params and add "ac -2" to custom audio options

**Comment by cattface** [comment]
> This can already be accomplished by using "Audio Encoder AAC", just enable custom ffmpeg params and add "ac -2" to custom audio options

Thanks - however, after some further research there's a useful thread [on SuperUser ](https://superuser.com/questions/852400/properly-downmix-5-1-to-stereo-using-ffmpeg) which recommends downmixing with Robert Collier's Nightmode Dialogue rather than just the ac -2. 

This has the following parameters: 
`ffmpeg -i "sourcetrack.dts" -c dca -af "pan=stereo|c0=c2+0.30*c0+0.30*c4|c1=c2+0.30*c1+0.30*c5" "stereotrack.dts" 
`
How would I adapt for Unmanics custom FFmpeg paramaters? 

Many thanks!

**Comment by DualScorch** [comment]
Using `-af pan=stereo|c0=c2+0.30*c0+0.30*c4|c1=c2+0.30*c1+0.30*c5` in custom audio options seems to do the trick for me :). You can't use the "-c dca" part (dts encoder) since aac is hardcoded into the plugin.

**Comment by andreas-glaser** [comment]
Yes those custom commands work but will it also trigger a check and run/skip on already converted files which are not yet downmixed? I would assume we still need a plugin to do that?

**Comment by cattface** [comment]
> Using `-af pan=stereo|c0=c2+0.30*c0+0.30*c4|c1=c2+0.30*c1+0.30*c5` in custom audio options seems to do the trick for me :). You can't use the "-c dca" part (dts encoder) since aac is hardcoded into the plugin.

Thanks! Tested and working for me.
